src/storage/aggregator.cc

```diff
@@ -205,14 +214,48 @@ bool Aggregator::Update(const std::string& key, const std::string& row, const ui
     if (window_type_ == WindowType::kRowsNum) {
         aggr_buffer.ts_end_ = cur_ts;
     }
-    bool ok = UpdateAggrVal(base_row_view_, row_ptr, &aggr_buffer);
+    bool ok = false;
+    if (reverse) {
+        ok = UpdateAggrValAfterDeleteRange(base_row_view_, row_ptr, &aggr_buffer);
+    } else {
+        ok = UpdateAggrVal(base_row_view_, row_ptr, &aggr_buffer, reverse);
```

Comment:
why pass the `reverse` argument that is always `false`

Reply:
Because UpdateAggrVal will be called in other places, and the paremeter 'reverse' in other places may be true